Phonebook.Frontend/docker-compose.yml

```
@@ -18,7 +18,7 @@ services:
       context: ./
       dockerfile: Dockerfile
     environment:
-      DEBUG: 'true'
+      # DEBUG: 'true'
```

das hatte ich wohl ausversehen raus genommen, kannst du es wieder einkomentieren?
```suggestion
      DEBUG: 'true'
```